content/docs/deployment/csmoperator/drivers/powerscale.md

```diff
@@ -1,144 +1,152 @@
-   | ***Common parameters for node and controller*** |
-   | CSI_ENDPOINT | The UNIX socket address for handling gRPC calls | No | /var/run/csi/csi.sock |
-   | X_CSI_ISI_SKIP_CERTIFICATE_VALIDATION | Specifies whether SSL security needs to be enabled for communication between PowerScale and CSI Driver | No | true |
-   | X_CSI_ISI_PATH | Base path for the volumes to be created | Yes | |
-   | X_CSI_ALLOWED_NETWORKS | Custom networks for PowerScale export. List of networks that can be used for NFS I/O traffic, CIDR format should be used | No | empty |
-   | X_CSI_ISI_AUTOPROBE | To enable auto probing for driver | No | true |
-   | X_CSI_ISI_NO_PROBE_ON_START | Indicates whether the controller/node should probe during initialization | Yes | |
-   | X_CSI_ISI_VOLUME_PATH_PERMISSIONS | The permissions for isi volume directory path | Yes | 0777 |
-   | ***Controller parameters*** |
-   | X_CSI_MODE   | Driver starting mode  | No | controller |
-   | X_CSI_ISI_ACCESS_ZONE | Name of the access zone a volume can be created in | No | System |
-   | X_CSI_ISI_QUOTA_ENABLED | To enable SmartQuotas | Yes | |
-   | ***Node parameters*** |
-   | X_CSI_MAX_VOLUMES_PER_NODE | Specify the default value for the maximum number of volumes that the controller can publish to the node | Yes | 0 |
-   | X_CSI_MODE   | Driver starting mode  | No | node | 
-
-4.  Execute the following command to create PowerScale custom resource:
-    ```kubectl create -f <input_sample_file.yaml>``` .
-    This command will deploy the CSI-PowerScale driver in the namespace specified in the input YAML file.
-
-5.  [Verify the CSI Driver installation](../#verifying-the-driver-installation)
-    
-**Note** : 
-   1. "Kubelet config dir path" is not yet configurable in case of Operator based driver installation.
+---
+title: PowerScale
+linkTitle: "PowerScale"
+description: >
+  Installing Dell CSI Driver for PowerScale via Dell CSM Operator
+---
+
+## Installing CSI Driver for PowerScale via Dell CSM Operator
+
+The CSI Driver for Dell PowerScale can be installed via the Dell CSM Operator.
+To deploy the Operator, follow the instructions available [here](../../#installation).
+
+Note that the deployment of the driver using the operator does not use any Helm charts and the installation and configuration parameters will be slightly different from the one specified via the Helm installer.
+
+**Note**: MKE (Mirantis Kubernetes Engine) does not support the installation of CSI-PowerScale via Operator.
+
+### Listing installed drivers with the ContainerStorageModule CRD
+User can query for all Dell CSI drivers using the following command:
+`kubectl get csm --all-namespaces`
+
+
+### Prerequisite
+
+1. Create namespace.
+   Execute `kubectl create namespace test-isilon` to create the test-isilon namespace (if not already present). Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'test-isilon'.

```

Comment:
Should we still be using references to "isilon"?

Reply:
not sure. @rajkumar-palani?